requirements/requirements.txt

```
@@ -8,7 +8,7 @@ hydra-core==1.3.2
 id-marl-eval @ git+https://github.com/instadeepai/marl-eval
 jax==0.4.30
 jaxlib==0.4.30
-jaxmarl
+jaxmarl @ git+https://github.com/RuanJohn/JaxMARL@unpin-jax
```

Just so we're clear we don't change anything else

```suggestion
jaxmarl @ git+https://github.com/RuanJohn/JaxMARL@unpin-jax  # only unpins the Jax version
```